### PR TITLE
Enable Background & Download Settings

### PR DESCRIPTION
**Pull request by dylangerdaly**
Enables Background and Download within Settings

### CODE REVIEW

File: src/main/kotlin/app/revanced/patches/layout/ForceSettingsPatch.kt [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import org.jf.dexlib2.Opcode
+
+private val compatiblePackages = listOf(
+    PackageMetadata(
+        "com.google.android.youtube",
+        listOf("17.14.35")
+    )
+)
+
+class ForceSettingsPatch : Patch(
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/layout/ForceSettingsPatch.kt:22]: This patch should have a different name. Something like "DownloadPatch" or "DownloadAndBackgroundPlaybackPatch".

File: src/main/kotlin/app/revanced/patches/layout/ForceSettingsPatch.kt [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                PatternScanMethod.Fuzzy(0), // FIXME: Test this threshold and find the best value.
+                compatiblePackages,
+                "Signature for the method required to be patched.",
+                "0.0.1"
+            ),
+            "Z",
+            AccessFlags.PUBLIC or AccessFlags.STATIC,
+            listOf("L", "L"),
+            listOf(
+                Opcode.IF_EQZ,
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/layout/ForceSettingsPatch.kt:44]: Are you sure this pattern is unique?

File: src/main/kotlin/app/revanced/patches/layout/ForceSettingsPatch.kt [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        "Force Download and Background Playback Settings Patch",
+        "Enable Download and Background Playback within Settings.",
+        compatiblePackages,
+        "0.0.1"
+    ),
+    listOf(
+        MethodSignature(
+            MethodSignatureMetadata(
+                "force-settings-patch-manager",
+                MethodMetadata(null, null), // unknown
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/layout/ForceSettingsPatch.kt:34]: Please leave a name of the method it matches. The name and class should be from the app version, in which this signature is valid/ has been created from

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]
There might also be a conflicting patch for background playback, you should remove either one with this PR.

**Comment by dylangerdaly** [comment]
Let me get Download working first, then I'll redo this

**Pull request closed without merging**